Libplanet.Tests/Blockchain/BlockChainTest.cs

```diff
@@ -686,6 +686,7 @@ public void AppendThrowsInvalidBlockBytesLengthException()
                     nonce: nonce,
                     privateKey: signer);
                 heavyTxs.Add(heavyTx);
+                nonce += 1;
```

Comment:
How pass this test before? 🤔

Reply:
Because we didn't check `.TxHash`, but re-calculate them. 😅